src/main/java/com/cloudbees/hudson/plugins/folder/computed/FolderComputation.java

```diff
@@ -110,11 +109,16 @@ public void run() {
             folder.updateChildren(listener);
             _result = Result.SUCCESS;
         } catch (InterruptedException x) {
+            LOGGER.log(Level.FINE, "recomputation of " + folder.getFullName() + " was aborted", x);
             listener.getLogger().println("Aborted");
             _result = Result.ABORTED;
         } catch (Exception x) {
-            // TODO skip stack trace for AbortException
-            x.printStackTrace(listener.fatalError("Failed to recompute children of " + getDisplayName()));
+            LOGGER.log(Level.FINE, "recomputation of " + folder.getFullName() + " failed", x);
+            if (x instanceof AbortException) {
+                listener.fatalError(x.getMessage());
+            } else {
+                x.printStackTrace(listener.fatalError("Failed to recompute children of " + folder.getFullDisplayName()));
```

Comment:
@jglick Why are you using `folder.getFullDisplayName()` and `folder.getFullName()`? Both return the same?